api/osrd_infra/schemas/infra.py

```diff
@@ -241,6 +241,7 @@ class Signal(BaseObjectTrait, TrackLocationTrait):
     installation_type: Optional[str]
     value: Optional[str]
     side: Side = Field(Side.CENTER, description="Side of the signal on the track")
+    default_aspect: str = Field("", description="Aspect displayed when no train is around")
```

Comment:
Why did you put a default value?
Is this field optional? In this case you should adapt the type to `Optional[str]`